iphone/Classes/TiErrorController.m

```diff
@@ -42,243 +35,22 @@ - (void)loadView
-                                   attribute:NSLayoutAttributeLeading
-                                   relatedBy:NSLayoutRelationEqual
-                                      toItem:view
-                                   attribute:NSLayoutAttributeLeading
-                                  multiplier:1
-                                    constant:20];
-  NSLayoutConstraint *centerConstraint4 =
-      [NSLayoutConstraint constraintWithItem:view
-                                   attribute:NSLayoutAttributeTrailing
-                                   relatedBy:NSLayoutRelationEqual
-                                      toItem:centerView
-                                   attribute:NSLayoutAttributeTrailing
-                                  multiplier:1
-                                    constant:20];
-  NSLayoutConstraint *centerConstraint5 =
-      [NSLayoutConstraint constraintWithItem:centerView
-                                   attribute:NSLayoutAttributeTop
-                                   relatedBy:NSLayoutRelationGreaterThanOrEqual
-                                      toItem:titleLabel
-                                   attribute:NSLayoutAttributeBottom
-                                  multiplier:1
-                                    constant:10];
-  NSLayoutConstraint *centerConstraint6 =
-      [NSLayoutConstraint constraintWithItem:disclosureLabel
-                                   attribute:NSLayoutAttributeTop
-                                   relatedBy:NSLayoutRelationGreaterThanOrEqual
-                                      toItem:centerView
-                                   attribute:NSLayoutAttributeBottom
-                                  multiplier:1
-                                    constant:10];
-
-  [centerView addConstraint:messageConstraint1];
-  [centerView addConstraint:messageConstraint2];
-  [centerView addConstraint:messageConstraint3];
-  [centerView addConstraint:buttonConstraint1];
-  [centerView addConstraint:buttonConstraint2];
-  [centerView addConstraint:buttonConstraint3];
-  [centerView addConstraint:buttonConstraint4];
-
-  [[self view] addConstraint:titleConstraint1];
-  [[self view] addConstraint:titleConstraint2];
-  [[self view] addConstraint:titleConstraint3];
-  [[self view] addConstraint:centerConstraint1];
-  [[self view] addConstraint:centerConstraint2];
-  [[self view] addConstraint:centerConstraint3];
-  [[self view] addConstraint:centerConstraint4];
-  [[self view] addConstraint:centerConstraint5];
-  [[self view] addConstraint:centerConstraint6];
-  [[self view] addConstraint:disclosureContraint1];
-  [[self view] addConstraint:disclosureContraint2];
-  [[self view] addConstraint:disclosureContraint3];
-
-  [dismissButton addTarget:self action:@selector(dismiss:) forControlEvents:UIControlEventTouchUpInside];
+  
+  UIScrollView* view = [[[NSBundle mainBundle] loadNibNamed:@"ErrorScreen" owner:self options:nil] objectAtIndex:0];
```

Comment:
Are we using an xib for convenience? Usually we try to add the AutoLayout code programmatically to have more control over it, but I understand that this may be critical for more complex layouts.